cdc_replication_test.py

```diff
@@ -95,6 +95,16 @@ def collect_data_for_analysis(self,
                                                    consistency_level=ConsistencyLevel.QUORUM, fetch_size=1000))
                 write_cql_result(res, os.path.join(self.logdir, 'replica-table'))
 
+    def upload_or_print(self, name: str, file_path: str, upload_dst: str) -> None:
+        self.log.info('Uploading {} to s3...'.format(name))
+        s3_url = S3Storage().upload_file(file_path=file_path, dest_dir=upload_dst)
+        if s3_url:
+            self.log.info('Uploaded {} to s3. Url: {}'.format(name, s3_url))
+            return
+
+        self.log.warning('Failed to upload {} to s3. Printing to stdout:'.format(name))
+        print_file_to_stdout(file_path)
+
```

Comment:
@fruch / @bentsi is there a built-in way to add the log to collect logs?

Reply:
I tried to look for one, but it seems that the set of collected logs is hardcoded.

Reply:
I didn't want to do that because this `log_entities` list is global (every test uses it) and `replicator.log` is specific to this one CDC test. Maybe a better solution would be to add some API that you can call from the test e.g. `self.add_log_file(path)` so different tests can add different log files.

Reply:
It is a good idea. But for that moment, collecting log is different stage, which is running independently. It needs test id only.
If such log is missing(will not find) it jus skipped it.